lib/commands/exam.js

```diff
@@ -343,15 +383,27 @@ module.exports = TestCommand.extend({
 
     events['set-modules-queue'] = function (modules) {
       //When `load-balance` option is valid we want to have one static list of modules on server side to send a module path to browsers.
-      const proto = Object.getPrototypeOf(this);
-      if (!proto.moduleQueue) {
+      const proto = Object.getPrototypeOf(this)
+      const browserId = (() => {
```

Comment:
Same comment as in the other file - you don't really need a function for this, do you?